index.bs

```diff
@@ -766,13 +766,17 @@ Participation in the Advisory Committee</h4>
 	(refer to the [=Member-only=] list
 	of <a href="https://www.w3.org/Member/ACList">current Advisory Committee representatives</a>. [[CURRENT-AC]])
 
-	When an organization joins W3C
-	(see “<a href="https://www.w3.org/membership/join/">How to Join W3C</a>” [[JOIN]]),
-	it <em class="rfc2119">must</em> name its Advisory Committee representative as part of their Membership Agreement.
+	As defined in the W3C Bylaws,
+	each W3C [=Member=] organization that is not a [=Member Association=] (known as a “<i>Consortium Member</i>” in the Bylaws)
```

Comment:
The existing parenthetical structure makes it seem that _a `Member Association` is known as a `Consortium Member` in the Bylaws_, rather than _a W3C Member organization that is not a `Member Association` is known as a `Consortium Member` in the Bylaws_.
```suggestion
	each “<i>Consortium Member</i>” (that is, each W3C [=Member=] organization that is not a [=Member Association=])
```

Reply:
or
```suggestion
	each W3C [=Member=] organization that is not a [=Member Association=] (that is, each organization the Bylaws would refer to as a “<i>Consortium Member</i>”)
```

Reply:
Good catch. I think I lean towards your first suggestion, but both seam reasonable to me. Any other opinions?